lib/decoding/luftdatenHandler.js

```diff
@@ -38,18 +38,34 @@ const { transformAndValidateMeasurements } = require('./validators'),
  *       }
  *     ]
  *   }
+ * Example in JSON PMS7003 with BME280
+ *   { "software_version": "NRZ-2017-099", 
+ *     "sensordatavalues":[
+ *       {"value_type":"PMS_P0","value":"1.00"},
+ *       {"value_type":"PMS_P1","value":"2.00"},
+ *       {"value_type":"PMS_P2","value":"2.00"},
+ *       {"value_type":"BME280_temperature","value":"24.98"},
+ *       {"value_type":"BME280_humidity","value":"39.60"},
+ *       {"value_type":"BME280_pressure","value":"100520.25"},
+ *       {"value_type":"samples","value":"580590"},
+ *       {"value_type":"min_micro","value":"216"},
+ *       {"value_type":"max_micro","value":"3503442"},
+ *       {"value_type":"signal","value":"-38"}
+ *     ]
+ *   }
  *
  * Example out JSON:
  * [
  *   {"sensor_id": "sensorID", "value":"value" },
  *   ...
  * ]
+ *
  */
 
 const matchings = {
-  p0: ['pm01', 'pm1.0'],
+  p0: ['pm01', 'pm0', 'p1.0', 'p0'],
```

Comment:
This object is used to match the json keys from the luftdaten json to the `title` of openSenseMap sensor definitons.
So if you define a sensor with the title `pm1.0`, the value should appear in the `p0` array. 
So the values as they were seem good enough for me.

Also, your changes directly below are not correct I think.